Backends/GLSL/BottomToGLSL.cpp

```diff
@@ -4379,7 +4379,10 @@ void gla::GlslTarget::emitGlaValueDeclaration(const llvm::Value* value, const ch
 
     const llvm::Constant* constant = llvm::dyn_cast<llvm::Constant>(value);
     if (constant) {
-        emitNamelessConstDeclaration(value, constant);
+        if (llvm::isa<llvm::PointerType>(value->getType()))
+            mapPointerExpression(value);
```

Comment:
Is this what made the structure tests start working again, even though they had nothing to do with interpolateAt*()?